sdk/version/version_base.go

```diff
@@ -13,5 +13,5 @@ var (
 
 	Version           = "1.12.0"
 	VersionPrerelease = "dev1"
-	VersionMetadata   = ""
+	VersionMetadata   = "ent"
```

Comment:
I'm not convinced this is a change we want here, but it was like this in the ENT repo. Perhaps we want to reset it to `""` here? I'm open to suggestions.

Reply:
Yup, this is an example of an OSS file that differs in the ENT repo. Maybe we should rectify this by having two separate files with build flags so they only get included in their respective (OSS vs ENT) builds?